### DISCUSSION TITLE
Where can I download the MMBench-test-CN and MMBench-dev-CN benchmark

### DISCUSSION THREAD

**Discussion by laserwave** [open-compass/opencompass#448] [Q&A] [answered]
Where can I download the MMBench-test-CN and MMBench-dev-CN benchmark

**Comment by kennymckormick** [comment, marked as answer]
Hi, @laserwave , 
We are preparing to release the dataset (basically, it's a translated Chinese version of MMBench with human verification). The dataset will be available (high likely) in next week.

**Reply by laserwave** [reply]
I am confused how the scores of MMBench-test(CN), MMBench-dev(CN) and CCBench-test on the leaderboard page （https://opencompass.org.cn/leaderboard-multimodal） are produced. Did the MLLMs use the same data? Or the results are produced by the official of OpenCompass?

**Reply by kennymckormick** [reply]
Hey, laserwave,
the MMBench-dev(CN) and MMBench-Test(CN) are now released at https://github.com/open-compass/MMBench. However, the CCBench (stands for Chinese Culture) is still not ready (will be released in this week). Besides, the evaluation service of MMBench-Test (CN) and CCBench is also not online yet. 
For now, you can test the MMBench-dev (CN) on your own and get the accuracy. But the evaluation of MMBench-Test (CN) and CCBench still requires our future release.

**Comment by shaojin-zhu** [comment]
> Hi, @laserwave , We are preparing to release the dataset (basically, it's a translated Chinese version of MMBench with human verification). The dataset will be available (high likely) in next week.

Hi, I was wondering if the CN dev/test datasets were ready to release?

**Reply by kennymckormick** [reply]
You can download the dataset here: 
https://github.com/open-compass/MMBench.

**Reply by shaojin-zhu** [reply]
Thank you so much for sharing.
When loading CN dataset, how should I set the 'sys_prompt'? which is "There are several options:" in english dataset.

Could I simply translate it as "有下列几个选项：" with the chinese colon.
May you add this instruction in the Readme file?
Thx

**Reply by shaojin-zhu** [reply]
Besides, I found the count of dataset after loading is slightly different:
MMBench_dev 4377
MMBench_CN_dev 4329
What did I do wrong?

------------------------------------------------------------------------------------
I found following index were missing in MMBench_CN_dev:
1222
135
156
226
233
584
595
791
1034
1069
1287
1759
1001222
1000135
1000156
1000226
1000233
1000584
1000595
1000791
1001034
1001069
1001287
1001759
2001222
2000135
2000156

**Reply by shaojin-zhu** [reply]
And I also found some text error in CN dataset, like in CN_test dataset:
index 245: "乒乓球从30¬∞角度或45¬∞角度发射后"

**Comment by laserwave** [comment]
@kennymckormick

**Reply by kennymckormick** [reply]
Resolved
